source/whats-new.txt

```diff
@@ -7,7 +7,7 @@ What's New
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 3
```

Comment:
Nit: The TOC seems really long now -- just thinking ahead, it'll only be longer as more versions come out. Maybe a depth of 2 instead of 3?

Reply:
I agree that it could get really long, but I think it's important to make the page easily navigable. I chose 3 to allow users to quickly find and jump to different things they may be interested in.

Reply:
A potential solution, if needed in the future, would be to convert the What's New TOC link to a drawer and have pages for each version. That is beyond the scope of this ticket, however.

Reply:
Agreed, this can be a future rework.